cmdb/framework/models/type.py

```diff
@@ -54,33 +54,86 @@ def to_json(cls, instance: "TypeSummary") -> dict:
         }
 
 
-class TypeSection:
-    __slots__ = 'type', 'name', 'label', 'fields'
+class TypeReference:
+    __slots__ = 'type_id', 'label', 'summaries', 'line', 'prefix', 'icon'
```

Comment:
If you changed the structure of the type refs, don't we need to update them with an updater for old refs? Correct me if I am wrong.